www/js/lib/settingsStore.js

```diff
@@ -62,10 +62,10 @@ define([], function () {
       localStorageTest = false;
     }
     // Now test for document.cookie API support
-    document.cookie = 'tempKiwixCookieTest=working;expires=Fri, 31 Dec 9999 23:59:59 GMT';
+    document.cookie = 'tempKiwixCookieTest=working;expires=Fri, 31 Dec 9999 23:59:59 GMT;samesite=Strict';
```

Comment:
@Abhirup-99 Please use the coding conventions / capitalization and spacing indicated in [MDN documentation](https://developer.mozilla.org/es/docs/Web/HTTP/Headers/Set-Cookie/SameSite), e.g. (this is their example):

`Set-Cookie: flavor=choco; SameSite=Secure`

So here you want a space after semicolon, and use correct capitalization. I know cookie headers are case-insensitive, but MDN is a respected source and it makes the code neater to use consistent conventions.